components/product-page/product-page.css

```diff
@@ -0,0 +1,155 @@
+@import url(/style.css);
+@import url(/components/color.css);
+:root{
+    --white-less:rgb(253, 249, 249);
+}
+
+.main-body{
+    margin-top:7rem;
+}
+.header-nav{
+    position: fixed;
+    z-index:2;
+    top:0;
+    width:100%;
+    box-shadow: 0px 15px 10px -15px #111;  
```

Comment:
use css variables for colors.

Reply:
i cannot see complete aside bar.
![Screenshot 2022-02-19 at 2 28 09 PM](https://user-images.githubusercontent.com/79623546/154794114-27a7de45-75e6-4f7e-99db-58b064bf7b45.png)

Reply:
I have made the changes